packages/rest/src/__tests__/integration/rest.application.integration.ts

```diff
@@ -163,6 +166,73 @@ describe('RestApplication (integration)', () => {
     await client.get(response.header.location).expect(200, 'Hi');
   });
 
+  context('mounting an express router on a LoopBack application', async () => {
+    givenApplication();
+
+    const router = express.Router();
+    await restApp.start();
+    client = createRestAppClient(restApp);
```

Comment:
Test setup belongs to a `before` or `beforeEach` hook.

That way the setup code is run only when the tests inside this `context` block are actually executed (think of `it.only` used elsewhere). More importantly, when the setup is executed from a hook, any errors are correctly associated with the relevant test case(s).